### DISCUSSION TITLE
Events Calendar in mantine-dates

### DISCUSSION THREAD

**Discussion by nabanita-sarkar** [mantinedev/mantine#839] [Feedback]
Is there any plan to add events calendar with month, week, day etc. views in mantine-dates?

**Comment by rtivital** [comment]
Maybe at some point

**Reply by nabanita-sarkar** [reply]
Awesome. And thank you for such an amazing library

**Reply by saipatro** [reply]
> Maybe at some point

Any update on this? 
Are we getting a week and month picker?
Are we getting this change in upcoming patch upgrades?

**Reply by rtivital** [reply]
No, there are no updates about this

**Reply by fordnox** [reply]
@rtivital i would like to offer sponsorship for implementation of Agenda, Day, Week, Month views [something similar to these](https://tailwindui.com/components/application-ui/data-display/calendars)

**Reply by rtivital** [reply]
No, thanks, we cannot accept any donations at this moment

**Reply by AliLotfi123** [reply]
@rtivital maybe you want to add it to roadmap for 2023?

**Reply by rtivital** [reply]
No, its is not a part of 2023 roadmap
